klasses/models.py

```diff
@@ -216,6 +220,13 @@ class BootcampRunEnrollment(TimestampedModel):
     bootcamp_run = models.ForeignKey(
         BootcampRun, on_delete=models.CASCADE, related_name="enrollments"
     )
+    change_status = models.CharField(
+        choices=ENROLL_CHANGE_STATUS_CHOICES, max_length=20, null=True, blank=True
+    )
+    active = models.BooleanField(
```

Comment:
Do we need to add any code to filter on `active=True`? I'm not sure if we are just storing the enrollment for now or if we are querying it somewhere.

Reply:
Good point, there so API query for enrollments yet but it would probably be best to add that filter [here](https://github.com/mitodl/bootcamp-ecommerce/blob/00dbe5378a3ee68643462ba3e93bab323ecc8519/ecommerce/api.py#L328).